cmake/UseGSettings.cmake

```diff
@@ -0,0 +1,23 @@
+# GSettings.cmake, CMake macros written for Marlin, feel free to re-use them.
+
+macro(add_schema SCHEMA_NAME)
```

Comment:
Please don't ship yet another GSettings CMake module with this PR. Please use the GSettings module from cmake-extras. Thanks.